2.0/db_models/named_section_embds.py

```
@@ -0,0 +1,40 @@
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, DateTime, func
```

Ruff (0.11.9)

`sqlalchemy.Text` imported but unused

Remove unused import: `sqlalchemy.Text`

(F401)